util/correspondingl1blocknumber.go

```diff
@@ -0,0 +1,34 @@
+// Copyright 2023-2024, Offchain Labs, Inc.
+// For license information, see:
+// https://github.com/offchainlabs/bold/blob/main/LICENSE.md
+
+package util
```

Comment:
I wonder if there isn't a better package than `util`.

It feels like it belongs in chain-abstraction to me. Or maybe in sol-implementation?

Reply:
Done.